examples/example-app-router-mixed-routing/src/middleware.ts

```diff
@@ -14,7 +14,9 @@ export default function middleware(request: NextRequest) {
   if (isAppRoute) {
     // Add a hint that we can read in `i18n.ts`
     request.headers.set('x-app-route', 'true');
-    return NextResponse.next({headers: request.headers});
+    return NextResponse.next({request: {
+      headers: request.headers
+    });
```

Comment:
Satisfy the linter and add a missing bracket …

```suggestion
    return NextResponse.next({request: {headers: request.headers}});
```